pages/docs/manual/latest/api/belt/array.mdx

```diff
@@ -765,7 +765,7 @@ let reduceWithIndex: (array<'a>, 'b, ('b, 'a, int) => 'b) => 'b
 
 `reduceWithIndex(xs, f)`
```

Comment:
Isn't this definition wrong as well?

```suggestion
`reduceWithIndex(arr, init, f)`
```
```suggestion
`reduceWithIndex(xs, f)`
```

Unrelated: always gets me that the array parameter is called `xs`... what does that even mean.

Reply:
Yes that is correct reduceWithIndex is also wrong. your suggestion looks way better